frontend/src/components/meta_popup.rs

```diff
@@ -99,24 +90,33 @@ impl Component for MetaPopup {
                     Input::Title => {
                         let target: HtmlInputElement = c.target_dyn_into().unwrap_throw();
                         self.meta.title = target.value();
-                        self.errors = self.errors.check(&self.meta.title, &self.meta.description);
+                        self.errors = self.errors.check(&self.meta.title, &self.meta.description, "");
                     }
                     Input::Desc => {
                         let target: HtmlTextAreaElement = c.target_dyn_into().unwrap_throw();
                         self.meta.description = target.value();
-                        self.errors = self.errors.check(&self.meta.title, &self.meta.description);
+                        self.errors = self.errors.check(&self.meta.title, &self.meta.description, "");
                     }
                 };
                 true
             }
-            Msg::ServerResponed(_) => {
+            Msg::ServerResponded(_) => {
                 self.send_pending = false;
                 ctx.props().on_close.emit(());
                 true
             }
         }
     }
 
+    fn changed(&mut self, ctx: &Context<Self>, _old_props: &Self::Properties) -> bool {
```

Comment:
not a fan of this function reordering, why did you move it below the `update`? it makes it hard to now see if the content of the function changed or just the place. so lets please reduce diffs in a PR to a minumum

Reply:
I have rebased the changes to undo that change.